test/adapter/json_api/links_test.rb

```diff
@@ -13,6 +13,10 @@ class LinkAuthorSerializer < ActiveModel::Serializer
             end
 
             link :other, '//example.com/resource'
```

Comment:
I don't feel strongly about this, although it is not clear from the discussion whether they are disallowing relative urls or protocol-relative urls.